tasks/main.yml

```diff
@@ -126,7 +126,22 @@
   copy:
     content: "{{ humio_permissions | default('{}') }}"
     dest: "{{ humio_data_path }}/{{ humio_host_id }}-{{ item.key }}/data/view-group-permissions.json"
+    mode: 0644
+    owner: humio
```

Comment:
I don't think it's a good idea to have Humio own the file as long as Humio can read the file.

Let's keep `root` as owner

Reply:
I removed that so it's just setting it to `0644` now. That said, what makes you hesitant to change the owner? All other files created by Humio are owned by the `humio` user. This permissions file doesn't contain passwords or other sensitive data (at least, not anything that's more sensitive than the log data that's stored on disk with `humio` as the owner). It just seems odd to me that we'd be setting the owner on everything else to `humio` but leave this one file that is definitely part of the Humio installation owned by root.

Reply:
Old habits from my side, maybe. But I mainly see the file as configuration, and I like to keep configuration as read only.